src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy

```diff
@@ -25,32 +25,47 @@ class MandateEmailSenderSpec extends Specification {
     @SpringBean
     MandateEmailService mandateEmailService = Mock(MandateEmailService)
 
+    @SpringBean
+    UserConversionService userConversionService = Mock(UserConversionService)
+
     def "send email when second pillar mandate event was received" () {
         given:
         User user = sampleUser().build()
+
+        UserPreferences contract = new UserPreferences()
```

Comment:
contract?

Reply:
"contract" taken from existing example. Changed to "userPreferences" throughout file